src/util/ens.js

```diff
@@ -0,0 +1,13 @@
+import ENS from "ethjs-ens";
+
+
+const ens = new ENS({ provider: window.web3.currentProvider, network: 1 });
```

Comment:
Huge, huge bug here!
Please make sure to also test on mobile and in incognito windows when developing. The website will completely break on this line if the user doesn't have a web3 wallet extension. 
To test on mobile you can add this to your start script in package.json
`"start": "react-scripts start --host 0.0.0.0",`
then after you run `npm run start` you can just connect to `<your commputer's local IP>:3000` from your mobile browser. 

Please read the dependency README: https://www.npmjs.com/package/ethjs-ens
It clearly says you should check for `typeof window.web3 !== 'undefined'` and otherwise you have to set your own provider. 

You could just handle it something like this in the ens.js file:
```javascript
import HttpProvider from 'ethjs-provider-http';

export const initENS = () => {
  if (typeof window.web3 !== 'undefined') {
    return new ENS({ provider: window.web3.currentProvider, network: 1 });
  } else {
    const provider = new HttpProvider('https://mainnet.infura.io/v3/f1d9de7ebcd04578b9f5696f04760115')
    return new ENS({ provider: provider, network: 1 });
  }
}

export const convertENS = async (ens, value) => {
    try {
      const result = await ens.lookup(value);
      return result
    } catch (e) {
      return value
    }
}
```

Then you can just store the ens resolver on the state of the main component.